modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ModelUtils.java

```diff
@@ -1126,7 +1134,10 @@ public static String getParentName(ComposedSchema composedSchema, Map<String, Sc
                     } else {
                         // not a parent since discriminator.propertyName is not set
                         hasAmbiguousParents = true;
-                        refedWithoutDiscriminator.add(parentName);
+                        Boolean isNotExtractedInlineSchema = !parentName.equals(schemaName+"_allOf");
```

Comment:
`boolean` instead of `Boolean`? Don't think we actually need an object here

Reply:
This has been changed to type boolean
True we don't need an object but having it named conveys that we are checking that a schema isNotExtractedInlineSchema which helps with understanding.